+++
abstract = """There is good evidence that an immune response to melanoma predicts better survival. We report a bioinformatic study designed to identify immune subgroups of primary melanomas in order to better understand the biology of immune cell/tumor interaction, and to identify candidate predictive biomarkers for immunotherapy.

We applied an approach described by Angelova et al. to the inference of tumor immune cell infiltrates using differential expression of signature genes, in 703 primary melanoma transcriptomes from the Leeds Melanoma Cohort (LMC). Consensus clustering of immune cell scores identified 3 subgroups in LMC and replication was sought in The Cancer Genome Atlas (TCGA) data set, (primarily metastatic samples). We then used the whole transcriptomic data to explore associated biological pathways: a protein-protein interaction network analysis of genes was performed in Cytoscape plugin Reactome FIVIZ. Nodal genes were identified and the cellular origin of the gene expression signature further explored using immunohistochemistry (IHC) and in patient-derived melanoma cell culture transcriptomes. Clinicopathological and environmental factors, and immune cell scores associated with survival within immune subgroups, were then analysed using a multivariate Cox proportional hazard model.

We identified three subgroups with transcriptomic patterns indicative of: low, intermediate and high immune cell infiltration. The groups were concordant with histological evidence of tumor infiltrating lymphocytes. Genes enriched in the low, compared with the high immune group, were in cell cycle, metabolic and Beta-catenin pathways. MYC was the nodal (the most influential) gene of the protein-protein interaction network in this group. In the high immune group, genes were enriched in immune pathways, including antigen processing and presentation, with the nodal gene - NFKB1. Based on the literature and our results, we hypothesised that MYC downregulates HLA-class I, hence we tested MYC correlations in the melanoma cell line transcriptomes (in which immune cells were absent). Indeed, MYC was negatively correlated with many HLA class I genes, with HLA-B having the strongest result: R=-0.57. IHC of primary tumors showed tumor nuclear expression of MYC. Reported smoking independently predicted a worse outcome for patients classified as the high immune group (HR=3.52, P=0.008). The transcriptomic analysis showed that smoking correlated with an increased Th2 cell score (P=0.03) and reduced NK56 bright score (P=0.01) in that group.

We report evidence that MYC plays an important role in immune evasion in primary melanoma. The transcriptomic data support in vitro findings that MYC downregulates HLA class I expression. Furthermore, we present important results that smoking status modifies immune response in melanoma and is an independent risk factor for melanoma death. We hypothesise that smoking may impact on immunotherapy efficacy."""
authors = ["Pozniak J", "Nsengimana J", "Laye J", "O'Shea S", "Schindler S", "Levesque M", "Bishop T", "Newton-Bishop J"]
date = 2018-07-01
doi = "10.1158/1538-7445.am2018-5204"
featured = false
highlight = true
math = true
publication = "*Tumor Biology* 2018; 78(Suppl 13):Abstract 5204"
publication_short = "*Tumor Biol* 2018; 78:Abstract 5204"
publication_types = ["1"]
summary = "*Tumor Biology* 2018; 78(Suppl 13):Abstract 5204"
tags = ["Nsengimana"]
title = "MYCexpression and smoking as drivers of failure of immune response to melanoma"
+++
